Comparing source compatibility of /tmp/japicmp/japicmp/target/japicmp-0.15.7.jar against /home/mmois/.m2/repository/com/github/siom79/japicmp/japicmp/0.15.6/japicmp-0.15.6.jar
===  UNCHANGED CLASS: PUBLIC japicmp.compat.Test  (default serialVersionUID changed)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
